src/Controller/Admin/ActivityController.php

```diff
@@ -213,6 +237,56 @@ public function deleteAction(Request $request, Activity $activity): Response
         ]);
     }
 
+    /**
+     * Archives an activity.
+     */
+    #[Route('/{id}/archive', name: 'archiveActivity')]
```

Comment:
The name of this route, as well as the route for `activateAction` does not follow the naming scheme used elsewhere (snake case and hierarchical with the top first). Instead, rename them to `archive` and `activate` (which will automatically be expanded to `admin_activity_archive`).